src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/net/GRPCConnectionManager.java

```diff
@@ -73,7 +72,7 @@
 
   public GRPCConnectionManager(final boolean shouldUseHttps) {
     this.shouldUseHttps = shouldUseHttps;
-    this.port = Util.RPC_PORT;
+    this.port = -1;
```

Comment:
Is there a reason you initialize to -1 instead of 0? 0 would make the default behavior the same as before this commit right?

Reply:
You are right. That was a miss on my part.